packages/asc-ui/src/components/Downloader/DownloaderContent/DownloaderContent.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
```

Comment:
Same, do you need to split it into download and downloader content?

Reply:
yes, the content should scale inside the main component